Project.toml

```diff
@@ -31,7 +31,7 @@ Colors = "0.12"
 Dates = "^1.9"
 EnergyModelsBase = "0.7 - 0.8"
 EnergyModelsGeography = "0.9 - 0.10"
-EnergyModelsInvestments = "0.6 - 0.7"
+EnergyModelsInvestments = "0.6 - 0.8"
```

Comment:
This change is irrelevant as 0.6 and 0.7 are not workable with `TimeStruct` 0.9. Neither is EMB 0.8.0 and EMG 0.10.0. The required changes for `TimeStruct` 0.9 do however no longer make it backwards compatible, *i.e.*, `osc` instead of `sc`. 
I would hence set the dependencies as

```toml
EnergyModelsBase = "0.8.1"
EnergyModelsGeography = "0.10.1"
EnergyModelsInvestments = "0.8"
```

That way, we do not allure the user to use wrong versions of these packages.

Similarly, I would increase to 

```toml
julia = "1.10"
```

as LTS. This is a prerequisite for all EMX packages nowadays. (except in packages where I forgot to include it...)